Dockerfile.local

```diff
@@ -1,7 +1,7 @@
 # Base image to use at runtime
```

Comment:
Ugh I hate that we have a duplicated Dockerfile. Should really be a separate build stage based off the production Dockerfile. Not your problem, though.